os/kernel/task/task_dumptasks.c

```diff
@@ -106,8 +109,10 @@ void task_show_tcbinfo(struct tcb_s *tcb)
 	lldbg("Sigdeliver  : %p\n", tcb->xcp.sigdeliver);
 #endif
 #ifdef CONFIG_LIB_SYSCALL
-	lldbg("Nsyscalls   : %u\n", tcb->xcp.nsyscalls);
-	lldbg("Syscall     : %p\n", tcb->xcp.syscall);
+	lldbg("Nsyscalls   : %u\n", nsyscall);
+	for (int i = 0; i < nsyscall; i++) {
+		lldbg("Syscall     : %p\n", tcb->xcp.syscall[i].sysreturn);
```

Comment:
How about printing index?
And what does it mean, index 0 and index 1?
what data is in index 0 and index 1?

Reply:
System call return address is stored in the syscall[] array.
It points to the next instruction after syscall is executed.
And, it seems that syscalls can be nested. So each index in the array indicates the nesting level.

@neel-samsung Please print the index also as requested by Chang.

Reply:
I have added the index in assert log as requested